PyInstaller/compat.py

```diff
@@ -28,7 +28,7 @@
 is_py3 = sys.version_info[0] == 3
 # Distinguish specific code for various Python versions.
 is_py27 = sys.version_info >= (2, 7) and sys.version_info < (3, 0)
-# PyInstaller supports only Python 3.3+
+# PyInstaller supports only Python 3.4+
 # Variables 'is_pyXY' mean that Python X.Y and up is supported.
 is_py34 = sys.version_info >= (3, 4)
```

Comment:
This can be removed, too.

Reply:
`is_py34` is used here:
https://github.com/pyinstaller/pyinstaller/blob/8e199ebac090435fc7ba54524b2238fc5d3a77d3/PyInstaller/compat.py#L172-L180

Should that be reversed into something like this?

```python
# In Python 3.4 module 'imp' is deprecated and there is another way how
# to obtain magic value.
if is_py27:
    # This fallback should work with Python 2.7.
    import imp
    BYTECODE_MAGIC = imp.get_magic()
else:
    # Python 3.4+
    import importlib.util
    BYTECODE_MAGIC = importlib.util.MAGIC_NUMBER
```

And shall we delete `is_py27` and replace it with `is_py2`?